.github/workflows/test.yml

```diff
@@ -43,6 +55,7 @@ jobs:
       - run: |
           chmod -R +x bin
           sudo rm /etc/containers/storage.conf
+      - run: |
```

Comment:
What is the reason for this change?

Reply:
My intention to not name the steps was to be able to see what is going to be executed from the jobs overview page. Unfortunately it only displays the first line from multi-line executions. By adding another run step, It now looks like this: 
![screenshot](https://user-images.githubusercontent.com/695473/105463319-30e08a80-5c90-11eb-84e1-d98bd3c61d49.png)

Seems more clear to me what is going on.